lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -49,7 +49,12 @@ class InfoCommand < HammerCLIForeman::InfoCommand
           Fields::Field.new(:label => _('Region'), :path => [:region])
         ],
         'libvirt' => [
-        ]
+        ],
```

Comment:
it looks good to me, but please check this review: 
https://github.com/theforeman/hammer-cli-foreman/pull/420
it would be great to have in each compute resource file, a function for 'provider_spesific_fileds' so that compute resource plugins will able to add their specific details easily.

Reply:
resolved!